src/lib/cache.js

```diff
@@ -128,6 +129,4 @@ class Cache {
   }
 }
 
-const cache = new Cache()
-
-module.exports = cache
+export const cache = new Cache()
```

Comment:
Not sure why, but babel did not like this being exported with `module.exports`, leading to some of the churn around cache imports throughout

Reply:
This file crosses the client/server boundary in subtle ways, so possibly related to that. In any case, better than `module.exports`, just kinda weird and annoying.